recipe/meta.yaml

```diff
@@ -23,12 +23,16 @@ requirements:
   run:
     - python
     - absl-py >=0.1.6
+    - astor >=0.6.0
     - enum34 >=1.1.6  # [py<34]
+    - gast >=0.2.0
+    - grpcio >=1.8.6
     - mock >=2.0.0    # [py27]
-    - numpy >=1.12.1
+    - numpy >=1.13.3
     - protobuf >=3.4.0
     - six >=1.10.0
-    - tensorboard >=1.5.0,<1.6.0
+    - tensorboard >=1.6.0,<1.7.0
+    - termcolor >=1.1.0
     - backports.weakref >=1.0rc1
```

Comment:
Although not mentioned in the setup.py file I believe `backports.weakref` is only required on Python <3.4.  That said I have no complaints about it being included as it doesn't hurt.

Reply:
Seems reasonable. Should be fixed. Thanks.